db/fdb_fend.c

```diff
@@ -4927,7 +4927,9 @@ int fdb_get_remote_version(const char *dbname, const char *table,
         flags = 0;
     }
 
-    snprintf(sql, sizeof(sql), "select table_version(\'%s\')", table);
+    int tot = snprintf(sql, sizeof(sql), "select table_version(\'%s\')", table);
```

Comment:
this is too low to check for user provided dbname length; if you're doing it here, make sure we pass back -3 (syntax error or similar)

Reply:
Well, I was just trying to get to protect against bad tablename being passed in. Since it will not be common at all, I figured generic error would suffice. Can we agree to leave it with generic or return FDB_ERR_FDB_TBL_NOTFOUND instead. -3 is not syntax error and, there is no other code that seems close to syntax error.

Reply:
Returning FDB_ERR_FDB_TBL_NOTFOUND.